src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,31 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        Reader reader = new Reader();
+        Writer writer = new Writer();
+        writer.createCsvFile(toFileName);
+        File readFile = new File(fromFileName);
+        File fromFile = new File(toFileName);
+        List<String> result = reader.readCsvFileWithData(readFile.toPath());
+        int buyCount = 0;
+        int supplyCount = 0;
+        int total;
```

Comment:
The 'total' variable is only used once and does not need to be declared separately. You can directly pass the calculation to the 'writeCsvFileWithData' method.